### ISSUE TITLE
Alter landing page

### ISSUE THREAD

**Issue by mrdrogdrog** [hedgedoc/react-client#125]
**Is your feature request related to a problem? Please describe.**
The landing page is hard coded in every codimd instance and has always the same content, that is advertising the features. The admin can't change the content without knowing react and recompiling the app. 

**Describe the solution you'd like**
We could offer the admin to place a markdown file in the public directory. The app can fetch this file, compile it with markdownit and show the output. If no file is present, it shows the default landing page.

**Describe alternatives you've considered**
- Leaving the hard coded content.
- The admin can alter the code and recompile the code

**Additional context**
This is low priority.

**Comment by ErikMichelson** [comment]
I'm unsure if we really want to make the landing page fully customizable. This is a bit like the discussion about full-support for custom themes.

But as we talked about banners, maybe an idea would be to allow admins to set a banner message (similar to GitLab) that will be displayed on the landing page if configured.

**Comment by mrdrogdrog** [comment]
One argument against this issue could be: We could create another possibility for XSS.
Except if we don't allow script, allow only images from the codimd backend, etc. etc. etc.

**Comment by mrdrogdrog** [comment]
But I still think it could be useful. And it's much safer than e.g. editing a pad and using it's code, because you must have write access to the public directory.

**Comment by mrdrogdrog** [comment]
Maybe we could add an "IKnowWhatImDoingModeByUsingThisFlag"-Mode like chrome/firefox/glxgears?

**Comment by SISheogorath** [comment]
Has that ever worked?

**Comment by mrdrogdrog** [comment]
No. lol. The idea of the flag was more of a joke :sweat_smile:

**Comment by mrdrogdrog** [comment]
I don't want to replace the complete landing page. Just the part under the "login or feature" button.

**Comment by DerMolly** [comment]
I'll work on a optional banner with a message on the landing page, now.

**Comment by SISheogorath** [comment]
Are we talking about a banner now or a full fledged content section on the landing page?

**Comment by mrdrogdrog** [comment]
I'm not sure?.. I think @DerMolly is talking about a banner. My opinion hasn't changed. I'm still thinking that it's better to replace the content on the landing page (below the buttons).
Or we could implement both.

**Comment by mrdrogdrog** [comment]
And regarding security: Replacing the content for the landing page isn't less secure than opening a doc. We would use MarkdownIt on both pages. If you think about it, it would be even MORE secure than the editor, because you need filesystem access for the landing page.

**Comment by mrdrogdrog** [comment]
BTW: An information banner is out of scope for this issue.
THIS issue is only about altering the landing page.

**Comment by DerMolly** [comment]
The question is do we want a info banner, dynamic content on the intro page or both?

**Comment by mrdrogdrog** [comment]
If you want a banner, then please open another issue

**Comment by ccoenen** [comment]
I would love to encourage admins to use this space in an inviting / informative manner. Of course they can choose to ignore this, but this very prominent space should be dedicated to one of these things. And by suggesting it and maybe even adding a pre-written template text, this can be encouraged even more:

* Who runs this instance
* What it is for
* If it's here "for good" or for testing
* Who is allowed/invited to write text here

We should also have basic usage info on this page for complete newbies. If admins choose to remove that, we can't stop them, but we can set sensible defaults.

**Comment by DerMolly** [comment]
After #563 is merged this could be done
